flutter-gui-tests/testSrc/io/flutter/tests/gui/InspectorTest.kt

```diff
@@ -0,0 +1,82 @@
+package io.flutter.tests.gui
+
+import com.intellij.testGuiFramework.fixtures.ActionButtonFixture
+import com.intellij.testGuiFramework.fixtures.IdeFrameFixture
+import com.intellij.testGuiFramework.framework.RunWithIde
+import com.intellij.testGuiFramework.framework.Timeouts
+import com.intellij.testGuiFramework.impl.GuiTestCase
+import com.intellij.testGuiFramework.impl.GuiTestUtilKt
+import com.intellij.testGuiFramework.impl.button
+import com.intellij.testGuiFramework.launcher.ide.CommunityIde
+import org.fest.swing.edt.GuiActionRunner.execute
+import org.fest.swing.edt.GuiQuery
+import org.fest.swing.fixture.JButtonFixture
+import org.fest.swing.timing.Pause
+import org.fest.swing.util.StringTextMatcher
+import org.junit.Assert.assertNotNull
+import org.junit.Test
+import java.awt.Container
+import java.awt.event.KeyEvent
+
+@RunWithIde(CommunityIde::class)
+class InspectorTest : GuiTestCase() {
+
+  @Test
+  fun importSimpleProject() {
+    // TODO(messick) killall Simulator
+    ProjectCreator.importProject()
+    ideFrame {
+      selectSimulator()
```

Comment:
It looks like this test is set up to work with the iOS Simulator on macos. We should plan to refactor to instead use the flutter test device (available on all platforms, w/o having to install any iOS or Android tooling).

Reply:
The equivalent of `flutter -d flutter-tester run`. We'll need to change the plugin to make that work. I'll take a look.